README.md

```diff
@@ -25,7 +25,7 @@ These tutorials demonstrate a variety of tasks, domains, labeling techniques, an
 
 # Installation
 
-To install Snorkel, we recommend using `pip`:
+To install Snorkel, we recommend using `pip` (with `python>=3.6`):
```

Comment:
Nit (since it's a req, not a rec): `Snorkel requires Python 3.6 or later. To install Snorkel, ...`

Reply:
👍